docs/usage/openapi/ui_plugins.rst

```diff
@@ -139,57 +139,6 @@ In the following example, we configure the OpenAPI root path to be ``/docs``:
 This will result in any of the OpenAPI endpoints being served at ``/docs`` instead of ``/schema``, e.g.,
 ``/docs/openapi.json``.
 
-Backward Compatibility
```

Comment:
I think we should keep this around and add it to a "Migrating to 3.0" doc. Every breaking change should be documented there, and ideally, we do it as we go, so we don't have to do it all at once like last time with 2.0 :eyes: